code/game/gamemodes/autotraitor/autotraitor.dm

```diff
@@ -1,7 +1,3 @@
-//This is a beta game mode to test ways to implement an "infinite" traitor round in which more traitors are automatically added in as needed.
-//Automatic traitor adding is complete pending the inevitable bug fixes.  Need to add a respawn system to let dead people respawn after 30 minutes or so.
```

Comment:
hmm yes, we should add a respawn system :clueless:

Reply:
It has been 12 years, maybe now is the time! 🤣